Tables/FE8 Nightmare modules/Class & Character editors/FE8 Character Editor.csv

```diff
@@ -208,50 +208,50 @@ Pegasus Messenger,0x263,0x2bb,0xcd,0x9,0x0,0x2,0,0x0,1,0,0,0,0,0,0,0,0,0,0,0,0,0
--0,0x267,0x0,0xed,0x41,0x0,0x0,0,0x0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
--0,0x267,0x0,0xee,0x41,0x0,0x0,0,0x0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
--0,0x267,0x0,0xef,0x41,0x0,0x0,0,0x0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
--0,0x267,0x0,0xf0,0x41,0x0,0x0,0,0x0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
--0,0x267,0x0,0xf1,0x41,0x0,0x0,0,0x0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
--0,0x267,0x0,0xf2,0x41,0x0,0x0,0,0x0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
--0,0x267,0x0,0xf3,0x41,0x0,0x0,0,0x0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
--0,0x266,0x2bd,0xf4,0x6d,0x0,0x5,0,0x0,1,0,0,0,0,2,0,0,0,0,0,0,0,0,0,0,0,85,25,60,10,15,15,25,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
--0,0x266,0x2bd,0xf5,0x6e,0x0,0x5,0,0x0,1,0,0,0,0,2,0,0,0,0,0,0,0,0,0,0,0,85,25,60,10,15,15,25,0x0,0x0,0x0,0x0,0x0,0x0,0x40,0x0,0x0,0x0,0x0,0x0,0x0,0x0
--0,0x266,0x2bd,0xf6,0x6f,0x0,0x5,0,0x0,1,0,0,0,0,2,0,0,0,0,0,0,0,0,0,0,0,85,25,60,10,15,15,25,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
--0,0x266,0x2bd,0xf7,0x70,0x0,0x5,0,0x0,1,0,0,0,0,2,0,0,0,0,0,0,0,0,0,0,0,85,25,60,10,15,15,25,0x0,0x0,0x0,0x0,0x0,0x0,0x40,0x0,0x0,0x0,0x0,0x0,0x0,0x0
--0,0x266,0x2bd,0xf8,0x6d,0x0,0x5,0,0x0,1,0,0,0,0,2,0,0,0,0,0,0,0,0,0,0,0,85,25,60,10,15,15,25,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
--0,0x265,0x2bc,0xf9,0x6e,0x0,0x5,0,0x0,1,0,0,0,0,2,0,0,0,0,0,0,0,0,0,0,0,85,25,60,10,15,15,25,0x0,0x0,0x0,0x0,0x0,0x0,0x40,0x0,0x0,0x0,0x0,0x0,0x0,0x0
--0,0x265,0x2bc,0xfa,0x71,0x0,0x5,0,0x0,1,0,0,0,0,2,0,0,0,0,0,0,0,0,0,0,0,85,25,60,10,15,15,25,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
--0,0x265,0x2bc,0xfb,0x72,0x0,0x5,0,0x0,1,0,0,0,0,2,0,0,0,0,0,0,0,0,0,0,0,85,25,60,10,15,15,25,0x0,0x0,0x0,0x0,0x0,0x0,0x40,0x0,0x0,0x0,0x0,0x0,0x0,0x0
--0,0x264,0x0,0xfc,0x71,0x0,0x5,0,0x0,1,0,0,0,0,2,0,0,0,0,0,0,0,0,0,0,0,85,25,60,10,15,15,25,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x3,0x0,0x0,0x0
--0,0x267,0x0,0xfd,0x41,0x0,0x0,0,0x0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
--0,0x268,0x0,0xfe,0x41,0x0,0x0,0,0x0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
--0,0x269,0x0,0xff,0x41,0x0,0x0,0,0x0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
+0,0x267,0x0,0xd1,0x41,0x0,0x0,0,0x0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0,0x0
```

Comment:
I'd prefer if thes -0's weren't changed, but it's not breaking or anything so I guess it's fine...